languageFactories/language-factory-core/src/main/scala/cromwell/languages/util/ImportResolver.scala

```diff
@@ -227,21 +245,37 @@ object ImportResolver {
         }).contextualizeErrors(s"download $toLookup")
       }
 
-    private def getUri(toLookup: WorkflowSource): Either[NonEmptyList[WorkflowSource], ResolvedImportBundle] = {
-      implicit val sttpBackend = HttpResolver.sttpBackend()
-      val responseIO: IO[Response[WorkflowSource]] = sttp.get(uri"$toLookup").headers(headers).send()
-
-      // temporary situation to get functionality working before
+    private def getUri(toLookup: WorkflowUrl): Checked[ResolvedImportBundle] = {
+      // Temporary situation to get functionality working before
       // starting in on async-ifying the entire WdlNamespace flow
-      val result: Checked[WorkflowSource] = Await.result(responseIO.unsafeToFuture(), 15.seconds).body.leftMap { e =>
-        NonEmptyList(e.toString.trim, List.empty)
+      // Note: this will cause the calling thread to block for up to 30 seconds
+      // (15 for the auth header lookup, 15 for the http request)
+      val unauthedAttempt = getUriInner(toLookup, Map.empty)
+      val result = if (StatusCodes.NotFound == unauthedAttempt.code) {
```

Comment:
[Yoda condition](https://en.wikipedia.org/wiki/Yoda_conditions)

Reply:
TIL about Yoda condition. Just to clarify, are you suggesting this be changed to the normal way? Or just pointing out that this is Yoda condition?

Reply:
I left the comment to obliquely request consistency, yeah. But certainly not the most urgent thing.

Reply:
Sounds good. I can revert the condition to make it consistent 👍

Reply:
My thinking was under the "avoiding null conditions" heading in that wiki page. I think `==` is null-safe in java anyway though so we would probably be fine either way. 

FWIW I don't fully agree with the page's argument against, but 🤷‍♂️ it's probably easier to read and equivalently safe so 👍 on refactoring